types/hapi__joi/hapi__joi-tests.ts

```diff
@@ -944,6 +944,9 @@ schema = Joi.link(str);
         value = schema.validate(value).value;
 
         result = schema.validate(value);
+        if (result.error) {
+            throw Error("error should not be set");
```

Comment:
Let's use single quotes to match the style of other strings

Reply:
fair enough. give me a second.